libV2/tasks/IngestShard.js

```diff
@@ -28,6 +29,52 @@ class IngestShardTask extends BaseTask {
         this._stripEventUUID = options.stripEventUUID !== undefined ? options.stripEventUUID : true;
     }
 
+    // eslint-disable-next-line class-methods-use-this
+    _registerMetricHandlers() {
+        const ingestedTotal = new promClient.Counter({
+            name: 'utapi_ingest_shard_task_ingest_total',
+            help: 'Number of metrics ingested',
+            labelNames: ['origin', 'containerName'],
+        });
+
+        const ingestedSlow = new promClient.Counter({
```

Comment:
In addition to this, we could store the sum of the ingestion lag (what is computed as `shardAge` if that makes sense), as a counter, this way we can easily query the average lag over a certain time range or other aggregating queries.